develop/develop-images/dockerfile_best-practices.md

```diff
@@ -541,11 +541,10 @@ FROM alpine
 ENV ADMIN_USER="mark"
 RUN echo $ADMIN_USER > ./mark
 RUN unset ADMIN_USER
-CMD sh
```

Comment:
alpine has `sh` as default for a while now, and it was not needed for the example